be/src/service/doris_main.cpp

```diff
@@ -308,7 +307,6 @@ struct Checker {
 
 int main(int argc, char** argv) {
     doris::signal::InstallFailureSignalHandler();
-    doris::init_signals();
     // create StackTraceCache Instance, at the beginning, other static destructors may use.
     StackTrace::createCache();
```

Comment:
The code is no longer needed, because ExecEnv manually `destroy()` at the end of `main()`

But there’s nothing wrong with keeping it

Reply:
> The code is no longer needed, because ExecEnv manually `destroy()` at the end of `main()`
> 
> But there’s nothing wrong with keeping it

We need this. 
`init_signals` is for sigint and sigterm

Reply:
> The code is no longer needed, because ExecEnv manually `destroy()` at the end of `main()`
> 
> But there’s nothing wrong with keeping it

and it should be moved to bottom of jvm init, or jvm's signal handler for sigint and sigterm will overwrite it